bin/configurable-http-proxy

```diff
@@ -87,7 +87,7 @@ cli
   )
   .option("--insecure", "Disable SSL cert verification")
   .option("--host-routing", "Use host routing (host as first level of path)")
-  .option("--metrics-ip <ip>", "IP for metrics server", "0.0.0.0")
+  .option("--metrics-ip <ip>", "IP for metrics server", "")
```

Comment:
I've verified that apiIp became accessible when configured with a blank string just as if it would have been configured with `0.0.0.0`, but I think using a blank string, we also acquire the sensible IPv4 and/or IPv6 compatibility we want.